src/main/java/edu/cornell/kfs/krad/antivirus/util/DummyAVUtils.java

```diff
@@ -0,0 +1,22 @@
+package edu.cornell.kfs.krad.antivirus.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.cornell.kfs.krad.antivirus.service.ScanResult.Status;
+
+public final class DummyAVUtils {
+
+    public static Status determineScanStatusFromScanResult(String result) {
+        switch (StringUtils.defaultString(result)) {
+            case "PASSED" :
+                return Status.PASSED;
+            case "FAILED" :
+                return Status.FAILED;
+            case "ERROR" :
+                return Status.ERROR;
+            default : 
+                return null;
```

Comment:
StringUtils.defaultString is null safe and returns an empty String should the input parameter to the static method (i.e. result) happen to be a null.  
I will have the default dummy implementation return value be Status.ERROR (similar to the real implementation) instead of null. Thanks for catching that.